docs/source/developing_dallinger_setup_guide.rst

```diff
@@ -67,8 +67,27 @@ If it does not return a number, you have not installed Postgres
 correctly in your ``/Applications`` folder or something else is horribly
 wrong.
 
-On Ubuntu, follow the instructions under the heading "Installation"
-`here <https://help.ubuntu.com/community/PostgreSQL>`__.
+Ubuntu users can install Postgres using the following instructions
+
+::
+
+    sudo apt-get update && apt-get install -y postgresql postgresql-contrib
+
+To run postgres, use the following command
+
+::
+
+    service postgresql start
+
+After that you'll need to run the following commands:
+::
+
+    runuser -l postgres -c "createuser -ds root"
+    createuser dallinger
+    createdb -O dallinger dallinger
+    sed /etc/postgresql/9.5/main/pg_hba.conf -e 's/md5/trust/g' --in-place
```

Comment:
It would be good either to make this line Postgres-version independent OR add some text telling people to replace `9.5` with their actual version. Ditto for the next line.

Reply:
Added a note.